### ISSUE TITLE
Deploy build error Prisma' has no exported member 'UserSelect'

### ISSUE THREAD

**Issue by julimancan** [chrishoermann/zod-prisma-types#191]
When deploying to vercel getting build error: 
![image](https://github.com/chrishoermann/zod-prisma-types/assets/64240466/0d261066-1d05-4c43-a740-6e7026865ae7)

this is my package.json: 
    "@prisma/client": "5.1.1",
    "prisma": "5.1.1",
    "zod": "3.21.1",
    "zod-prisma-types": "^2.7.7",

I just have a user model and I don't get any errors in localhost when I run npm run build or dev. Any idea what could be going wrong?

**Comment by chrishoermann** [comment]
@julimancan have you been able to fix this issue? if not can you provide your (simple) schema? in the screenshot it looks like there is a newer version of prisma client installed than you use in your package.json..

**Comment by julimancan** [comment]
Yes I was going through the different issues and saw some incompatibility issues which led me to try different versions of prisma but I keep getting the same errors. ```model User {
  id              String   @id @unique
  firstName       String
  lastName        String
  hasImage        Boolean  @default(false)
  image           String?
  address1        String?
  address2        String?
  city            String?
  stateProvince   String?
  postalCode      String?
  country         String?
  email           String   @unique
  phone           String?
  businessName    String?
  instagramHandle String?
  createdAt       DateTime @default(now())
  updatedAt       DateTime @updatedAt
  role            Role     @default(VENDOR)
}

enum Role {
  ADMIN
  VENDOR
}```
and yes this is my schema
